### PR TITLE
upgraded errai version

### PR DESCRIPTION
**Pull request by mbiarnes**

### CODE REVIEW

**Comment by mbiarnes** [comment]
https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/943
https://github.com/kiegroup/appformer/pull/673

File: pom.xml [hunk 1]
```diff
@@ -64,7 +64,7 @@
 
     <!-- Custom Freemarker build with workaround for random concurrent access issue (annotation processors). See AF-600 for more info.-->
     <version.org.freemarker>2.3.26.jbossorg-1</version.org.freemarker>
-    <version.org.jboss.errai>4.6.0-SNAPSHOT</version.org.jboss.errai>
+    <version.org.jboss.errai>4.7.0-SNAPSHOT</version.org.jboss.errai>
```
Comment (mareknovotny) [inline pom.xml]: again start to use 4.6.0.Final?

Comment (mbiarnes) [reply pom.xml]: fixed

**Comment by mbiarnes** [comment]
@mareknovotny do you think it is needed to do a FDB in this case?

**Comment by mareknovotny** [comment]
I don't think so

**Pull request merged**